src/state/selectors/searches.js

```diff
@@ -122,11 +122,13 @@ export const getSearchAnnotationsForWindow = createSelector(
 export const getSelectedContentSearchAnnotationIds = createSelector(
   [
     getWindow,
+    getSearchForCompanionWindow,
   ],
-  window => (window && window.selectedContentSearchAnnotation) || [],
+  (window, search) => (search && search.selectedContentSearchAnnotation)
```

Comment:
FWIW, I'd be easily persuaded that selected-for-search and selected-for-window are different and should have different selectors. I went back-and-forth on that and decided not to care (yet)

Reply:
👍 I think it's fine to make that distinction when it become necessary.